ntbs-service/wwwroot/source/Components/CascadingDropdown.ts

```diff
@@ -102,6 +103,17 @@ const CascadingDropdown = Vue.extend({
             values.forEach(item => optionInnerHtml += `<option value="${item.value}">${item.text}</option>`);
             return optionInnerHtml;
         },
+        setSelectToValueOrDefault: function (selectElement: HTMLSelectElement, targetValue: string) {
+            for (let i = 0; i < selectElement.options.length; i++) {
+                if (selectElement.options[i].value === targetValue) {
+                    selectElement.value = targetValue;
+                    return;
+                }
+            }
+
+            selectElement.value = "";
+            selectElement.dispatchEvent(new Event('change'));

```

Comment:
Why do we need this here, but not in the filtered dropdown?

Reply:
It's so that the second filtered list updates. Without this you could: change the TB service to Leeds then change the PHEC to something that isn't 'Yorkshire' which would remove the TB service but the case managers from Leeds would still remain.

Reply:
Great - I think it may be worth adding a comment saying that the purpose of this is to make sure that the case managers update

Reply:
(Although I guess you don't want to explicitly refer to the case managers, as this component could be used in other settings)